duckduckgo/iOS
# Alphabetize country list in geoswitching menu

Task/Issue URL: https://app.asana.com/0/1203137811378537/1206375204651916/f
Tech Design URL:
CC:

**Description**:

The order of the countries in the locations list is confusing. Let’s alphabetize it. Despite what one may think, this makes more sense on the app side as we’re only localizing for presentation, which is an app specific concern. Reason being that the country code is the source of truth for identifying a country. If the logic to alphabetize were more complex, there might be an argument to do it in BSK and share the code, but it’s only 1-2 lines and not worth the additional complexity to abstract it. Let me know if this needs more explanation.

**Steps to test this PR**:
1. Make sure VPN access is activated
2. Go to Settings -> Network Protection -> VPN Settings -> Preferred Location
3. **Make sure the countries are listed alphabetically.**

###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
